src/main/scala/firrtl/Compiler.scala

```diff
@@ -100,7 +102,9 @@ object CircuitState {
   * strictly supersets of the "lower" forms. Thus, that any transform that
   * operates on [[HighForm]] can also operate on [[MidForm]] or [[LowForm]]
   */
-@deprecated("CircuitForm will be removed in 1.3. Switch to Seq[TransformDependency] to specify dependencies.", "1.2")
+@deprecated(
+  "Mix-in the DependencyAPIMigration trait into your Transform and specify it's Dependency API dependencies. See: https://bit.ly/2Voppre",
```

Comment:
s/it's/its here and other deprecation warnings

Reply:
Good catch. Thanks for checking the text. Incorrect grammar in the deprecation message isn't great. :grimacing: Fixed in the rebase.